tests/D2L.CodeStyle.Analyzers.Test/Specs/ImmutabilityAnalyzer.cs

```diff
@@ -143,15 +152,15 @@ public class SomeMutableClassWithImmutableTypeParameter<[Immutable] T> { }
 
 		[ImmutableBaseClass]
 		public class SomeImmutableBaseClass { }
-		static SomeImmutableBaseClass FuncReturningSomeImmutableBaseClass() => null;
+		public static SomeImmutableBaseClass FuncReturningSomeImmutableBaseClass() => null;

```

Comment:
What I did was set the build action of this file to Compile and used that to clean up random diagnostics. Random changes like this were to fix stuff like that (in this case because there were visibility problems elsewhere with this being `private`.)